packages/prisma/schema.prisma

```diff
@@ -315,12 +315,23 @@ model Team {
   apiKeys              ApiKey[]
   credentials          Credential[]
   accessCodes          AccessCode[]
+  isOrganization       Boolean                 @default(false)
+  organizationSettings OrganizationSettings?
   instantMeetingTokens InstantMeetingToken[]
   pendingPayment       Boolean                 @default(false)
 
   @@unique([slug, parentId])
```

Comment:
Notice there is no unique constraint [slug, isOrganization] which was there in original PR